nginx_exporter.go

```diff
@@ -193,6 +193,9 @@ func main() {
 			</body>
 			</html>`))
 	})
-
+	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can you rename the path to `/-/healthy` to be consistent with prometheus and official exporters?